examples/example_requirements.txt

```diff
@@ -1,6 +1,7 @@
-pytket == 1.25.0
-pytket-cirq == 0.34.0
-pytket-qiskit == 0.48.0
-pytket-qujax == 0.17.0
+pytket == 1.26.0
+pytket-cirq == 0.35.0
+pytket-qiskit == 0.50.0
+pytket-qujax == 0.18.0
 openfermion 
 pytest
+pydantic == 2.6.4
```

Comment:
Why is this specific version of pydantic pinned?

Reply:
That was only for testing, remove it again in [33e069a](https://github.com/CQCL/pytket-docs/pull/310/commits/33e069a2a1e3300f9af430c4709e28eece005771)

Thank you for spotting!